console/network/src/consensus_heights.rs

```diff
@@ -88,14 +126,15 @@ pub(crate) fn load_test_consensus_heights<N: crate::Network>()
     // Check if we can read the heights from an environment variable.
     match std::env::var("CONSENSUS_VERSION_HEIGHTS") {
         Ok(height_string) => {
+            let parsing_error = format!("Expected exactly {NUM_CONSENSUS_VERSIONS} ConsensusVersion heights.");
             // Parse the heights from the environment variable.
-            let parsed_test_consensus_heights: [u32; crate::NUM_CONSENSUS_VERSIONS] = height_string
+            let parsed_test_consensus_heights: [u32; NUM_CONSENSUS_VERSIONS] = height_string
                 .replace(" ", "")
                 .split(",")
-                .map(|height| height.parse::<u32>().unwrap())
+                .map(|height| height.parse::<u32>().expect("Heights should be valid u32 values."))
                 .collect::<Vec<u32>>()
                 .try_into()
-                .unwrap();
+                .expect(&parsing_error);
```

Comment:
nit: why not just define the error message here directly?

Reply:
expect complained about using `&format!(...)` as argument